extension/Makefile

```diff
@@ -0,0 +1,32 @@
+IPYTHON_CONFIG := /nail/home/syedj/.ipython/profile_default
+
+.PHONY: all
+all: clean build develop
+
+.PHONY: develop-notebook
+develop-notebook:
+	venv/bin/jupyter notebook
+
+develop:
+	. venv/bin/activate
+	venv/bin/jupyter lab --watch
+.PHONY: build
+build: frontend-build
+	venv/bin/pip install -I .
+	venv/bin/jupyter labextension install
+	venv/bin/jupyter labextension enable sparkmonitor
+	venv/bin/jupyter serverextension enable --py sparkmonitor
+	ipython profile create && echo "c.InteractiveShellApp.extensions.append('sparkmonitor.kernelextension')" >>  $(IPYTHON_CONFIG)/ipython_kernel_config.py
```

Comment:
In particular, this line is used to enable the spark monitor kernel extension. This step was taken from the old instruction verbatim and I imagine I'll need to change this for installation with Jupyterhub; right now, I'm not sure how I'd go about doing that.

Reply:
I've updated the makefile so that it creates a local ipython profile and uses that to launch jupyter lab. However, whilst trying to install the extension into the core-ml docker image, the kernel extension still fails to be instantiated; I'm going to have to continue to look into this.